packages/desktop-client/src/components/reports/CategorySelector.tsx

```diff
@@ -26,30 +26,29 @@ export default function CategorySelector({
   return (
     <>
       <div>
-        <Button
-          type="bare"
-          style={{ padding: 4 }}
-          onClick={e => setUncheckedHidden(!uncheckedHidden)}
-        >
+        <Button onClick={() => setUncheckedHidden(state => !state)}>
           {uncheckedHidden ? (
             <>
               <Eye width={20} height={20} />
-              {'Checked'}
+              Show all categories
```

Comment:
Small nitpick: Maybe we should end both options with "categories" or both without?

Reply:
Adding `categories` to the other label makes the button super stretched out. Which didn't look as good.

Got any other suggestions how we could call these labels?

Reply:
How about `Show unchecked` and `Hide unchecked`?

Reply:
We can do that: https://github.com/actualbudget/actual/pull/1548/commits/fa7b25780e609ee79c67b603daf0a78d7a0cbaff